src/layer/Tooltip.js

```diff
@@ -413,9 +413,18 @@ Layer.include({
 
 
 	_openTooltip(e) {
-		if (!this._tooltip || !this._map || (this._map.dragging && this._map.dragging.moving())) {
+		if (!this._tooltip || !this._map) {
 			return;
 		}
+
+		// If the map is moving, we will show the tooltip after it's done.
+		if (this._map.dragging && this._map.dragging.moving()) {
+			this._map.once('moveend zoomend', () => {
+				this._openTooltip(e);
+			});
```

Comment:
This is the right approche but everytime the tooltip wil be opend it adds a new `once` listener